### ISSUE TITLE
[QUESTION] Chapter 12 - Wrong subsection name

### ISSUE THREAD

**Issue by Iulian277** [ageron/handson-ml2#575]
Hi! In the book at the `Chapter 12` there is a subsection called `Custom Training Loops`, but in the notebook is wrongly called `Computing Gradients Using Autodiff` (in the notebook are 2 sections with this name).

**Comment by ageron** [comment]
Good catch, thanks @Iulian277 ! 👍 
I fixed the notebook.
